compiler/rustc_hir/src/intravisit.rs

```diff
@@ -291,9 +292,9 @@ pub trait Visitor<'v>: Sized {
     }
 
     /// Invoked to visit the body of a function, method or closure. Like
-    /// visit_nested_item, does nothing by default unless you override
-    /// `nested_visit_map` to return other than `None`, in which case it will walk
-    /// the body.
+    /// `visit_nested_item`, does nothing by default unless you override
+    /// `Self::NestedFilter` to a type other than `None`, in which case
```

Comment:
```suggestion
    /// `Self::NestedFilter`.
```
and stop the doc here. The description on `visit_nested_item`should be enough.